cedar-policy/CHANGELOG.md

```diff
@@ -31,21 +33,21 @@ Cedar Language Version: 4.0
   in 3.2.0. This involved API breaking changes in both. See #757 and #854.
 - Moved `<PolicyId as FromStr>::Err` to `Infallible` (#588, resolving #551)
 - Removed unnecessary lifetimes from some validation related structs (#715)
-- Changed policy validation to reject comparisons and conditionals between
+- (*) Changed policy validation to reject comparisons and conditionals between
   record types that differ in whether an attribute is required or optional. (#769)
-- Changed JSON schema parser so that `Set`, `Entity`, `Record`, and `Extension`
+- (*) Changed JSON schema parser so that `Set`, `Entity`, `Record`, and `Extension`
```

Comment:
Note that we plan to undo this change (#1070), but noting it anyways for consistency.